components/lib/stepper/StepperBase.js

```diff
@@ -107,7 +107,8 @@ export const StepperBase = ComponentBase.extend({
         __TYPE: 'Stepper',
         activeStep: 0,
         orientation: 'horizontal',
-        linear: false
+        linear: false,
+        onChangeStep: undefined
```

Comment:
```suggestion
        onChangeStep: null
```

We typically use NULL

Reply:
Thank you for your comment, I'll keep it in mind

Reply:
Thanks for the PR.